lib/order.rb

```diff
@@ -0,0 +1,37 @@
+
+class Order
+  attr_reader :id
+  attr_accessor :products, :customer, :fulfillment_status
+
+  VALID_ORDER = [:pending, :paid, :processing, :shipped, :complete]
+
+  def initialize(id, products, customer, fulfillment_status =:pending)
+    @id = id
+    @products = products
+    @customer = customer
+    if VALID_ORDER.include?(fulfillment_status)
+      @fulfillment_status = fulfillment_status
+    else
+      raise ArgumentError #puts a statement here?
```

Comment:
This works, but you can also include an error message to accompany the exception. 

```suggestion
      raise ArgumentError, "Invalid Fulfillment status #{fulfillment_status}"
```